problems/[230216] 221. Maximal Square.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/maximal-square
+ *
+ * @param {character[][]} matrix
+ * @return {number}
+ */
+const maximalSquare = function (matrix) {
+  // 푼 시간 3시간 넘음
+  let max = 0;
+
+  for (let index = 0; index < matrix.length; index++) {
+    let link = 0;
+    let maxLink = 0;
+
+    for (let i = 0; i < matrix[index].length; i++) {
+      if (matrix[index][i] === "1") {
+        link++;
+      } else {
+        link = 0;
+      }
+
+      maxLink = Math.max(link, maxLink);
+    }
+
+    if (maxLink <= max) {
+      max = Math.max(max, maxLink);
+      continue;
+    }
+
```

Comment:
```suggestion
    if (maxLink <= max) {
      max = Math.max(max, maxLink);
      continue;
    }
    if (maxLink - 1 < max) {
      continue;
    }
```

이 부분의 위는 `maxLink`를 구하는 부분 같고, 그 아래는 `maxLink`를 검증하는 부분 같은데, 그 사이에 이렇게 `continue` 혹은 `break`문이 모여있는게 보기 편할 것 같아요.